### ISSUE TITLE
zipdownload - Error with many files with the same name

### ISSUE THREAD

**Issue by cmnardi** [roundcube/roundcubemail#6301]
When the email has many attached files with the same name the zip file is created with just one of then.
In the zipdowload plugin the method `_create_displayname` try to resolve this problem but i found a bug at line `216` #

**Comment by thomascube** [comment]
Indeed, the name check is broken since commit f8537d02213c7 due to a typo.

**Comment by cmnardi** [comment]
I created a [pull request](https://github.com/roundcube/roundcubemail/pull/6302)
